include/ola/base/Macro.h

```diff
@@ -63,6 +63,30 @@
 #endif  // __GNUC__
 
 /**
+ * @def OLA_FALLTHROUGH
+ * @brief Mark switch cases as fallthrough when required
+ *
+ * @examplepara
+ *   @code
+ *   switch (cond) {
+ *     case 'foo':
+ *       doFoo();
+ *       //after foo fallthrough to bar because $REASON
```

Comment:
Can you fix the lint of the example by adding a space between // and after please.